spec/03-plugins/28-grpc-gateway/01-proxy_spec.lua

```diff
@@ -5,12 +5,15 @@ for _, strategy in helpers.each_strategy() do
 
   describe("gRPC-Gateway [#" .. strategy .. "]", function()
     local proxy_client
-
+    local service1
+    local route1
+    local plugin1
+    local bp
```

Comment:
`bp` is not used in later tests so it should not be exposed, but rather local to `lazy_setup`.